src/pudl/transform/phmsagas.py

```diff
@@ -0,0 +1,159 @@
+"""Classes & functions to process PHMSA natural gas data before loading into the PUDL DB."""
+
+import pandas as pd
+from dagster import AssetIn, asset
+
+import pudl.logging_helpers
+from pudl.helpers import (
+    standardize_na_values,
+    standardize_phone_column,
+    zero_pad_numeric_string,
+)
+from pudl.metadata.dfs import POLITICAL_SUBDIVISIONS
+
+logger = pudl.logging_helpers.get_logger(__name__)
+
+##############################################################################
+# Constants required for transforming PHMSAGAS
+##############################################################################
+
+YEARLY_DISTRIBUTION_OPERATORS_COLUMNS = {
+    "columns_to_keep": [
+        "report_date",
+        "report_number",  # not in pudl/metadata/fields.py
+        "report_submission_type",  # not in pudl/metadata/fields.py
+        "report_year",
+        # None of the columns below are in pudl/metadata/fields.py
+        "operator_id_phmsa",
+        "operator_name_phmsa",
+        "office_address_street",
+        "office_address_city",
+        "office_address_state",
+        "office_address_zip",
+        "office_address_county",
+        "headquarters_address_street",
+        "headquarters_address_city",
+        "headquarters_address_state",
+        "headquarters_address_zip",
+        "headquarters_address_county",
+        "excavation_damage_excavation_practices",
+        "excavation_damage_locating_practices",
+        "excavation_damage_one_call_notification",
+        "excavation_damage_other",
+        "excavation_damage_total",
+        "excavation_tickets",
+        "services_efv_in_system",
+        "services_efv_installed",
+        "services_shutoff_valve_in_system",
+        "services_shutoff_valve_installed",
+        "federal_land_leaks_repaired_or_scheduled",
+        "percent_unaccounted_for_gas",
+        "additional_information",
+        "preparer_email",
+        "preparer_fax",
+        "preparer_name",
+        "preparer_phone",
+        "preparer_title",
+    ],
+    "columns_to_convert_to_ints": [
+        "report_year",
+        "report_number",
+        "operator_id_phmsa",
+        "excavation_damage_excavation_practices",
+        "excavation_damage_locating_practices",
+        "excavation_damage_one_call_notification",
+        "excavation_damage_other",
+        "excavation_damage_total",
+        "excavation_tickets",
+        "services_efv_in_system",
+        "services_efv_installed",
+        "services_shutoff_valve_in_system",
+        "services_shutoff_valve_installed",
+    ],
+    "capitalization_exclusion": ["headquarters_address_state", "office_address_state"],
```

Comment:
`preparer_email` should also get excluded here, emails are case insensitive

Reply:
Updated